enterprise/internal/insights/store/insight_store.go

```diff
@@ -792,6 +792,29 @@ func (s *InsightStore) HardDeleteSeries(ctx context.Context, seriesId string) er
 	return s.Exec(ctx, sqlf.Sprintf(hardDeleteSeries, seriesId))
 }
 
+func (s *InsightStore) GetUnfrozenInsightCount(ctx context.Context) (globalCount int, totalCount int, err error) {
+	rows := s.QueryRow(ctx, sqlf.Sprintf(getUnfrozenInsightCountSql))
+	if err = rows.Scan(
+		&globalCount,
+		&totalCount,
+	); err != nil {
+		return
+	}
+	return
+}
```

Comment:
All of these empty returns look a little odd, maybe we can simplify?

```suggestion
func (s *InsightStore) GetUnfrozenInsightCount(ctx context.Context) (globalCount int, totalCount int, err error) {
	rows := s.QueryRow(ctx, sqlf.Sprintf(getUnfrozenInsightCountSql))
	err = rows.Scan(
		&globalCount,
		&totalCount,
	)
	return
}
```

Reply:
Yes, good call!